tv.plex.PlexHTPC.yml

```diff
@@ -0,0 +1,109 @@
+  - --filesystem=xdg-download:create
+  - --filesystem=xdg-cache:rw
+  - --filesystem=xdg-config:rw
+  - --filesystem=xdg-data:rw
+  - --filesystem=home:ro
+  - --talk-name=org.freedesktop.PowerManagement
+  - --talk-name=org.freedesktop.ScreenSaver
+modules:
+  - name: plex-icons
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID
+      - install -Dm664 $FLATPAK_ID.png /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+    sources:
+      - type: dir
+        path: .
+
+  - name: libjpeg # with libjpeg.so.8
+    buildsystem: cmake-ninja
+    config-opts: &libjpeg_config_opts
+      - -DCMAKE_SKIP_RPATH:BOOL=YES
+      - -DENABLE_STATIC:BOOL=NO
+      - -DWITH_JPEG8:BOOL=YES
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib # uses lib64 by default
+    sources: &libjpeg_sources
+      - type: archive
+        url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/refs/tags/2.1.3.tar.gz
+        sha256: dbda0c685942aa3ea908496592491e5ec8160d2cf1ec9d5fd5470e50768e7859
+
+  - name: libevent
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DEVENT__LIBRARY_TYPE=SHARED
+    sources:
+      - type: git
+        url: https://github.com/libevent/libevent/
+        branch: release-2.1.12-stable
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/cmake
+
+  - name: libwebp
+    buildsystem: autotools
+    config-opts: [--disable-static, --enable-libwebpmux, --enable-libwebpdemux]
+    cleanup: [/share, /bin]
+    sources:
+      - type: archive
+        url: https://github.com/webmproject/libwebp/archive/refs/heads/0.5.1.zip
+        sha256: 55c5abdf0f153b26f89ad593181f0d7ff7aec03d6d51777598fe8d8c63a1eea0
+
+  - name: libsnappy
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DBUILD_SHARED_LIBS=ON"
+      - "-DSNAPPY_BUILD_TESTS=OFF"
+      - "-DCMAKE_BUILD_TYPE=Release"
+    sources:
+      - type: git
+        url: https://github.com/google/snappy
+        tag: 1.1.7
+
+  - name: minizip
+    subdir: contrib/minizip
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://zlib.net/zlib-1.2.12.tar.gz
+        sha256: 91844808532e5ce316b3c010929493c0244f3d37593afd6de04f71821d5136d9
+        x-checker-data:
+          type: anitya
+          project-id: 5303
+          stable-only: true
+          url-template: https://zlib.net/zlib-$version.tar.gz
+      - type: shell
+        commands:
+          - GLOBIGNORE=contrib && rm -rf *
+          - GLOBIGNORE=contrib/minizip && rm -rf contrib/*
+          - rm -f contrib/minizip/Makefile
+          - autoreconf -fiv contrib/minizip
+    cleanup:
+      - '*.la'
+
+  - name: plex-binaries
+    buildsystem: simple
+    build-commands:
+      - cp -a * /app
+    sources:
+      - type: archive
+        url: https://artifacts.plex.tv/plex-htpc-experimental/1.18.0.3028-34988b94/linux/PlexHTPC-1.18.0.3028-34988b94-linux-x86_64.tar.bz2
```

Comment:
they don't have a aarch64?

Reply:
Not at the moment. We might add it in the future.

Reply:
this is disappointing